### PR TITLE
Make test_current_inventory pylon compliant

### PR DESCRIPTION
**Pull request by andrewO423**

### CODE REVIEW

File: tests/test_current_inventory.py [hunk 1]
```diff
@@ -1,11 +1,10 @@
+"""This file provides the tests for the current_inventory class"""
```
Comment (stevemac1900) [inline tests/test_current_inventory.py]: Remove

File: tests/test_current_inventory.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 from available_inventory import InventoryQueries
 from inventory import Inventory
 
-import pytest
-
-base_dir = 'tests/sample_csv_files/'
 
 def make_beer(quantity):
+    """Makes a beer object to be used for the file tests"""
```
Comment (stevemac1900) [inline tests/test_current_inventory.py]: Remove

File: tests/test_current_inventory.py [hunk 3]
```diff
@@ -14,25 +13,31 @@ def make_beer(quantity):
 
 
 def make_queries(filename):
+    """Tests that a data file can be turned into an InventoryQuery"""
```
Comment (stevemac1900) [inline tests/test_current_inventory.py]: This one isn't actually testing anything, not sure whether it needs a docstring but if so it should probably change to better fit the functionality

File: tests/test_current_inventory.py [hunk 4]
```diff
@@ -14,25 +13,31 @@ def make_beer(quantity):
...
+    base_dir = 'sample_csv_files/'
     return InventoryQueries(Inventory(base_dir + filename))
 
 
 def test_no_inventory():
+    """Tests that no inventory file adds no inventory"""
```
Comment (stevemac1900) [inline tests/test_current_inventory.py]: Remove

File: tests/test_current_inventory.py [hunk 5]
```diff
@@ -14,25 +13,31 @@ def make_beer(quantity):
...
 
 def test_single_item_with_positive_quantity():
+    """Tests that a file with a single beer adds one beer to inventory"""
     expected = [make_beer(1)]
     queries = make_queries('single_positive_quantity.csv')
     assert expected == queries.get_available_inventory()
 
 
 def test_single_item_with_zero_quantity():
+    """Tests that one beer type with no quantity will generate empty inventory"""
```
Comment (stevemac1900) [inline tests/test_current_inventory.py]: Might be good to change the wording here to the same form as test_single_item_with_negative_quantity's docstring

**Comment by jpvonelli** [comment]
This looks fine, however this pylint message comes up: 
C: 25, 0: Function name "test_single_item_with_positive_quantity" doesn't conform to snake_case naming style (invalid-name).

This is probably because of the length of the name.  If so, I think we can ignore it.

**Pull request closed without merging**